src/Internal/Metadata/Cip25/Cip25String.purs

```diff
@@ -51,6 +51,8 @@ derive newtype instance ToData Cip25String
 derive newtype instance FromData Cip25String
 derive newtype instance EncodeAeson Cip25String
 
+derive instance newtypeCip25String :: Newtype Cip25String _
```

Comment:
We can't allow a Newtype instance, because it breaks the invariant guaranteed by `mkCip25String` constructor. You are seemingly using it for `wrap` and `unwrap`, just use pattern matching and the constructor

Reply:
done